lib/honeybadger/config.rb

```diff
@@ -277,7 +277,6 @@ def load_plugin?(name)
     end
 
     def insights_enabled?
-      return false if defined?(::Rails.application) && ::Rails.const_defined?("Console")
```

Comment:
`::Rails.const_defined?("Console")` was always a bit finnicky to me to determine if we are in a console or not. I'm opting to just remove it for now until there is a better way to detect it automatically.